hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -20,11 +20,12 @@
 
 /**
  * This class contains constants for Recon related configuration keys used in
- * SCM & Datanode.
+ * SCM &amp; Datanode.
  */
 public final class ReconConfigKeys {
 
-  /**
+  /**This class contains constants for Recon related configuration keys used in
+   * SCM &amp; Datanode.
```

Comment:
Why do we need to repeat the class description?

Javadoc comment can be converted to regular code comment on private constructor.

Reply:
Done